material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex

```diff
@@ -1,116 +1,240 @@
-\providecommand{\isolatedBuild}[1]{#1}% fallback definition lets this file build normally
-\isolatedBuild
-{
-\documentclass[11pt,letterpaper]{book}
-\usepackage{import}
-
-% This file must be found via the TEXINPUTS environment variable.
-\input{handout_helpers.tex}
-
-\pagestyle{plain}
-\pagenumbering{arabic}
-
-\begin{document}
-\HandoutHeader{Vector Equations and Geometry}
-%
-\vspace{-0.5pc}
-\small
-%
-\begin{enumerate}
-\item \textColorBold{darkerBlue}{Cameras and Robotics in the NFL}
-\\[0.5pc]
+%   File: FirstAndTen.tex
+% Author: Adam Leeper
+%------------------------------------------------------------------------------
+%\\[0.45pc]
+\providecommand{\isolatedBuild}[1]{#1}% Fallback definition to build normally.
+\isolatedBuild{
+  \documentclass[11pt,letterpaper]{book}
+  \input{handout_helpers.tex}   % Found via TEXINPUTS environment variable.
+  \isolatedBuildHeader{Vector Equations and Geometry}
+                      {Cameras and Robotics in the NFL}
 }
+%%%
+%%%
+%%%
+\newcommand\MyLBrace[2]{%
+  \left.\rule{0pt}{#1}\right\}\text{#2}}
```

Comment:
Can't define newcommands in these problem files, or they won't work when I `\input` a bunch of them into a single document.

Also, I don't think we need this.